lib/new_game.rb

```diff
@@ -16,72 +16,89 @@ def main_menu
 
   def play
     build_board
-    computer_place_ships
+    num_ships
+
+
+    @player_ships, @computer_ships = create_ships(@ship_count)
     player_place_ships
-    until (@player_cruiser.sunk? && @player_submarine.sunk?) || (@computer_cruiser.sunk? && @computer_submarine.sunk?)
+    computer_place_ships
+
+    until @player_ships.all?(&:sunk?) || @computer_ships.all?(&:sunk?)
       take_turns
     end
     end_of_game
   end
 
+  def num_ships
+    puts "Enter the number of ships you want to create:"
+    @ship_count = gets.chomp.to_i
+    if @ship_count < 1 || @ship_count.nil?
+    puts 'Invalid Input. Must create at least one ship.'
+      num_ships
+    end
```

Comment:
When we have time for refactoring, we can consider removing the global @ from ship_count and instead make the num_ships method return ship_count at the end.